src/main/java/com/codegym/shopyy/controller/UserController.java

```diff
@@ -0,0 +1,71 @@
+package com.codegym.shopyy.controller;
+
+import com.codegym.shopyy.model.dto.UserDto;
+import com.codegym.shopyy.payload.request.SearchRequest;
+import com.codegym.shopyy.service.ISecurityService;
+import com.codegym.shopyy.service.IUserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@CrossOrigin(value = "*", maxAge = 3600)
+@RestController
+@RequestMapping("/api/user")
```

Comment:
change to /api/roles to adapt REST standard

Reply:
resolved